internal/ethapi/api.coreth.go

```diff
@@ -0,0 +1,118 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ethapi
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/coreth/params"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+)
+
+const (
+	nAVAX = 1_000_000_000
+
+	minBaseFee       = params.EtnaMinBaseFee // 1 nAVAX
+	maxNormalBaseFee = 100 * nAVAX
+
+	minGasTip       = 1 // 1 wei
+	maxNormalGasTip = 20 * nAVAX
+
+	slowFeeNum = 19 // 19/20 = 0.95
+	fastFeeNum = 21 // 21/20 = 1.05
+	feeDen     = 20
```

Comment:
nit: I think it makes sense to use full Numerator / Denominator for these names.
slowFeeNum can bring "slowFeeNumber" to mind, even though it's kind of obvious.